samples/dnn/models.yml

```diff
@@ -241,14 +241,13 @@ googlenet:
 # Semantic segmentation models.
 ################################################################################
 
-fcn8s:
+fcnresnet50:
   load_info:
-    url: "http://dl.caffe.berkeleyvision.org/fcn8s-heavy-pascal.caffemodel"
-    sha1: "c449ea74dd7d83751d1357d6a8c323fcf4038962"
-  model: "fcn8s-heavy-pascal.caffemodel"
-  config: "fcn8s-heavy-pascal.prototxt"
-  mean: [0, 0, 0]
-  scale: 1.0
+    url: "https://github.com/onnx/models/raw/491ce05590abb7551d7fae43c067c060eeb575a6/validated/vision/object_detection_segmentation/fcn/model/fcn-resnet50-12.onnx"
+    sha1: "1bb0c7e0034038969aecc6251166f1612a139230"
+  model: "fcn-resnet50-12.onnx"
+  mean: [103.5, 116.2, 123.6]
+  scale: 0.019
```

Comment:
`sample` is missing.

Reply:
@asmorkalov Line 254 [https://github.com/gursimarsingh/opencv/blob/6314b2cb5c3e2c9c25be8382bfaa82b08acf4c86/samples/dnn/models.yml#L254](#254)